server/routes/referrals/referralFormPresenter.ts

```diff
@@ -47,7 +47,12 @@ export default class ReferralFormPresenter {
         },
         {
           title: 'Select desired outcomes',
-          url: this.calculateTaskUrl('desired-outcomes', this.taskValues.relevantSentence),
+          url: this.calculateTaskUrl(
+            this.referral.serviceCategoryIds && this.referral.serviceCategoryIds.length > 0
```

Comment:
I think whether the url is displayed or not should still depend on the relevant-sentence being completed?

My understanding is that for single referrals the serviceCategoryIds is a static single list that is created before the page exists. 



For cohort referrals the serviceCategoryIds are set by a new selection page on this referral form.

Reply:
I guess it depends whether you consider this a cohort or single referral - which is the ticket I'm working on to differentiate it.

Reply:
Resolved - since at this stage the referral could be either single or cohort. The logic makes sense for cohort.

Reply:
Yep, this is just for single service referrals at the moment - this was very much an early step (just added because the compiler complained that `this.referral.serviceCategoryIds[0]` could be `null`. I assumed this would be picked up in the future piece of work to update the form for cohort referrals.